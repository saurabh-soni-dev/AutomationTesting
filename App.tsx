import {View, Text} from 'react-native';
import React from 'react';

const App = () => {
  return (
    <View>
      <Text>First snapshot test</Text>
    </View>
  );
};

export default App;
