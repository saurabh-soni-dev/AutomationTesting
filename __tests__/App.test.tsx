import {render} from '@testing-library/react-native';
import App from '../App';

test('SNAPSHOT OF APP ', () => {
  const {toJSON} = render(<App />);
  expect(toJSON()).toMatchSnapshot();
});
