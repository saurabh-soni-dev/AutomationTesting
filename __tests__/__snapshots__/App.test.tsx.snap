// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SNAPSHOT OF APP  1`] = `
<View>
  <Text>
    First snapshot
  </Text>
</View>
`;
